### PR TITLE
Changed minimum size to gzip response & added missing test

### PR DESCRIPTION
**Pull request by zemariamm**
if you’re compressing files that are smaller than the MTU size of a TCP packet, you’re wasting your time. 1500 bytes is the MTU size for the internet since that is the largest size allowed at the network layer. If you take a file that is 1300 bytes and compress it to 800 bytes, it’s still transmitted in that same 1500 byte packet regardless, so you’ve gained nothing. That being the case, you should restrict the gzip compression to files with a size greater than a single packet, 1400 bytes (1.4KB) is a safe value.

### CODE REVIEW

File: gzip.go [hunk 1]
```diff
@@ -28,9 +28,10 @@ const (
 	// The examples seem to indicate that it is.
 	DefaultQValue = 1.0
 
-	// DefaultMinSize defines the minimum size to reach to enable compression.
-	// It's 512 bytes.
-	DefaultMinSize = 512
+	// For starters, if you’re compressing files that are smaller than the MTU size of a TCP packet, you’re wasting your time. 1500 bytes is the MTU size for the internet since that is the largest size allowed at the network layer.
```
Comment (jprobinson) [inline gzip.go]: Mind if we drop the `For starters,...` prefix and start this comment with `1500 bytes is the MTU...`?

**Comment by zemariamm** [comment]
@jprobinson no problemo! It was actually a "copy-paste" from a forum

**Comment by jprobinson** [comment]
thanks again, @zemariamm!

**Comment by jprobinson** [comment]
Sorry, cached github UI fail. I'll merge the proper commit in a sec.

**Comment by zemariamm** [comment]
@jprobinson my pleasure

**Pull request closed without merging**